crates/network/src/any/unknowns.rs

```diff
@@ -146,6 +146,11 @@ impl alloy_consensus::Transaction for UnknownTypedTransaction {
         self.gas_price().or(self.max_priority_fee_per_gas()).unwrap_or_default()
     }
 
+    fn is_dynamic_fee(&self) -> bool {
+        self.fields.get_deserialized::<U128>("maxFeePerGas").is_some()
+            || self.fields.get_deserialized::<U128>("maxFeePerBlobGas").is_some()
```

Comment:
can we use the `self.ty` here?

Reply:
how? I thought ab it too, but it would be incorrect for eg deposit tx to do smthg like `self.ty() > 1`

Reply:
oh, yeah we'd only catch known types, which defeats the purpose here